packages/evm/vitest.config.browser.mts

```diff
@@ -0,0 +1,16 @@
+
+import { configDefaults, defineConfig, mergeConfig } from 'vitest/config'
+import baseConfig from '../../config/vitest.config.browser.mts'
+
+export default mergeConfig(
+  baseConfig,
+  defineConfig({
+    test: {
+      exclude:      [
+        ...configDefaults.exclude,
+        // readDirSync method not provided fs mock for vite
+        'test/precompiles/eip-2537-bls.spec.ts',
```

Comment:
Guess this can be removed again (?) after BLS merge

Reply:
Update: just tested and realized it's another thing

Reply:
Yep, there are probably several of these where the mock that vitest injects a polyfill that's not fully mapping to the node builtin we're referencing.